--- a/include/onnxruntime/core/common/logging/logging.h
+++ b/include/onnxruntime/core/common/logging/logging.h
@@ -63,6 +63,11 @@ using Timestamp = std::chrono::time_point<std::chrono::system_clock>;
 // define a temporary value to determine whether to use the std::chrono or date implementation.
 #define ORT_USE_CXX20_STD_CHRONO __cplusplus >= 202002L

+// GCC added full support for std::chrono::operator<< in version 13.
+#if defined(__GNUC__) && !defined(__clang__) && __GNUC__ < 13
+#undef ORT_USE_CXX20_STD_CHRONO
+#endif
+
 // Apply constraints for mac builds
 #if __APPLE__
 #include <TargetConditionals.h>
